actions/webhook.go

```diff
@@ -20,16 +20,16 @@ type Webhook struct {
 }
 
 type WebhookEventInfo struct {
-	EventType     string            `json:"event_type"`
-	EventTime     string            `json:"event_time"`
-	ActionName    string            `json:"action_name"`
-	HookID        string            `json:"hook_id"`
-	RepositoryID  string            `json:"repository_id"`
-	BranchID      string            `json:"branch_id"`
-	SourceRef     string            `json:"source_ref,omitempty"`
-	CommitMessage string            `json:"commit_message"`
-	Committer     string            `json:"committer"`
-	Metadata      map[string]string `json:"metadata,omitempty"`
+	EventType      string            `json:"event_type"`
+	EventTime      string            `json:"event_time"`
+	ActionName     string            `json:"action_name"`
+	HookID         string            `json:"hook_id"`
+	RepositoryID   string            `json:"repository_id"`
+	BranchID       string            `json:"branch_id"`
+	SourceRef      string            `json:"source_ref,omitempty"`
+	CommitMessage  string            `json:"commit_message"`
+	Committer      string            `json:"committer"`
+	CommitMetadata map[string]string `json:"metadata,omitempty"`
```

Comment:
the json tag should probably also be renamed to `commit_metadata`

Reply:
Done.

Reply:
The JSON tag should be aligned?